pkg/packagekit/updates.jsx

```diff
@@ -301,7 +314,7 @@ function UpdateHistory(props) {
 class ApplyUpdates extends React.Component {
     constructor() {
         super();
-        this.state = { percentage: 0, timeRemaining: null, curStatus: null, curPackage: null };
+        this.state = { percentage: 0, timeRemaining: null, actions: [] };
```

Comment:
Indeed. I also broke the long lines a bit, see the [interdiff](https://paste.fedoraproject.org/paste/ss0h9Sn8okMExWpmp3kSdQ) to the previous version of this PR. Pushed.